Unix/installbuilder/datafiles/Linux.data

```diff
@@ -141,6 +144,15 @@ RemoveOmiService() {
 }
 
 ConfigureOmiService() {
+
+# If the environment variable OMI_DISABLE_SERVICE_START is set to a non-null value,
+# OMI will not be configured with service manager. This may be used in a container
+# environment, where service manager does not work reliably.
+
+    if [ "$OMI_DISABLE_SERVICE_START" ]; then
+        echo "Skipping start up of OMI ..."
+        return 1
```

Comment:
`return 0` here to be safe. We don't want a non-zero exit code "leaking out" to calling script.